### PR TITLE
(545) Chore: readme house keeping

### PR DESCRIPTION
**Pull request by mec**
## Changes

Spliting things out and adding documentation.

https://trello.com/c/MmMDrIVA

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [ ] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

**Review by jacksonj04 — APPROVED** [review]
Love me some documentation improvements, ship it!

File: README.md [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+[DfE Technical Guidance](https://technical-guidance.education.gov.uk/).
+
+## Shared components and patterns
+
+- [DfE GOV.UK Components](https://govuk-components.netlify.app/)
+- [DfE GOV.UK Form Builder](https://govuk-form-builder.netlify.app/)
+- [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend)
+- [GOV.UK Accessible autocomplete](https://github.com/alphagov/accessible-autocomplete)
+- [MOJ Frontend](https://github.com/ministryofjustice/moj-frontend)
+>>>>>>> 70394f8 (Update readme)
```
Comment (jkempster34) [inline README.md:15]: There is a Git merge addition thing here (don't know what its called)

File: doc/workflow-files.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Worklow files
+
+## Introduction
+
+We made the decision early on to define the task list that makes up the heart of
+our user journey in a yaml file.
```
Comment (jkempster34) [inline doc/workflow-files.md:6]: Maybe "in YAML" is more accurate, because its soon to be multiple files?

File: doc/users-accounts.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# User accounts
+
+## Introduction
+
+Our users sign in to access the applicaiton.
```
Comment (jkempster34) [inline doc/users-accounts.md:5]: Spelling of application

File: doc/users-accounts.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+The authentication is handled by the DfE Active Directory via Omniatuth.
+
+We store registered users in a local table, and authorise them to perform tasks
```
Comment (jkempster34) [inline doc/users-accounts.md:9]: Could "local table" just be "database table" or something. I think "local table" might have meaning outside of what this is saying

**Review by jkempster34 — CHANGES_REQUESTED** [review]
I would find this really useful when joining 😎 

Noticed a Git thing, and a couple of spelling things.

**Review by jkempster34 — APPROVED** [review]

**Pull request merged**